soh/soh/GameMenuBar.cpp

```diff
@@ -201,12 +194,110 @@ namespace GameMenuBar {
+                    }
+
+                    int fpsSlider = 1;
+                    if (currentFps == 20) {
+                        ImGui::Text("FPS: Original (20)");
+                    } else {
+                        ImGui::Text("FPS: %d", currentFps);
+                        if (currentFps == 30) {
+                            fpsSlider = 2;
+                        } else { // currentFps == 60
+                            fpsSlider = 3;
+                        }
+                    }
+                    if (CVarGetInteger("gMatchRefreshRate", 0)) {
+                        UIWidgets::DisableComponent(ImGui::GetStyle().Alpha * 0.5f);
+                    }
+
+                    if (ImGui::Button(" - ##WiiUFPS")) {
+                        fpsSlider--;
+                    }
+                    ImGui::SameLine();
+                    ImGui::SetCursorPosX(ImGui::GetCursorPosX() - 7.0f);
+
+                    UIWidgets::Spacer(0);
+
+                    ImGui::SliderInt("##WiiUFPSSlider", &fpsSlider, 1, 3, "", ImGuiSliderFlags_AlwaysClamp);
+
+                    ImGui::SameLine();
+                    ImGui::SetCursorPosX(ImGui::GetCursorPosX() - 7.0f);
+                    if (ImGui::Button(" + ##WiiUFPS")) {
+                        fpsSlider++;
+                    }
+
+                    if (CVarGetInteger("gMatchRefreshRate", 0)) {
+                        UIWidgets::ReEnableComponent("");
+                    }
+                    if (fpsSlider > 3) {
+                        fpsSlider = 3;
+                    } else if (fpsSlider < 1) {
+                        fpsSlider = 1;
+                    }
+
+                    if (fpsSlider == 1) {
+                        currentFps = 20;
+                    } else if (fpsSlider == 2) {
+                        currentFps = 30;
+                    } else if (fpsSlider == 3) {
+                        currentFps = 60;
+                    }
+                    CVarSetInteger("gInterpolationFPS", currentFps);
+                    SohImGui::RequestCvarSaveOnNextTick();
+                #else
+                    bool matchingRefreshRate =
+                        CVarGetInteger("gMatchRefreshRate", 0) && SohImGui::WindowBackend() != SohImGui::Backend::DX11;
+                    UIWidgets::PaddedEnhancementSliderInt(
+                        (currentFps == 20) ? "FPS: Original (20)" : "FPS: %d",
+                        "##FPSInterpolation", "gInterpolationFPS", minFps, maxFps, "", 20, true, true, false, matchingRefreshRate);
+                #endif
+                    UIWidgets::Tooltip(
+                        "Uses Matrix Interpolation to create extra frames, resulting in smoother graphics. This is purely "
+                        "visual and does not impact game logic, execution of glitches etc.\n\n"
+                        "A higher target FPS than your monitor's refresh rate will waste resources, and might give a worse result.\n\n"
```

Comment:
this only applies on directx where the slider isn't limited to the monitor refresh rate. part of me feels like we should conditionally add this line and part of me feels like it's not worth it. i guess the real thing is to make sure we remove this line if/when we get the match refresh rate stuff working on dx

Reply:
Yeah that's honestly the reason why I left it in here. It wouldn't be too terribly hard to have 2 tooltips depending on DX or not, but I don't think it's a super critical to leave it in here.

Reply:
I'm happy to keep it or remove it, your call